Introduction
------------

Predator assemblages can have strong top-down effects, both on community
structure and ecosystem processes (Estes et al. 2011). The strength of
these effects is often dependent on species composition, and can be more
or less than what simply the sum of effect sizes of individual predators
(Sih, Englund, and Wooster 1998; Ives, Cardinale, and Snyder 2005).
Detailed studies manipulating predator assemblages have identified a
great diversity of direct and indirect mechanisms underlying the effect
of predator diversity on communities. For example, predators differ in
their prefernces for microhabitats resulting in spatial or temporal
partitioning between predator species in their effects on prey and
synergistic effects on net predation rates (Schmitz 2007). When
predators do co-occur in a local patch, they may feed on different prey
species and so have additie top-down effects (Schmitz 2007). Conversely,
antagonistic predator-predator interactions can reduce the top-down
effects of a diverse consumer assemblage, for example when predators
feed directly on each other (intra-guild predation) or modify the
behaviour of predators or prey via non-consumptive (ie trait-mediated)
interactions (Sih, Englund, and Wooster 1998; Griswold and Lounibos
2006; Nyström et al. 2001). While there are many mechanisms potentially
underlying predator compositional effects, we lack a means of predicting
*a priori* which combinations are likely to have antagonistic versus
synergistic effects on ecosystem functions.

Several authors have hypothesized that the phylogenetic diversity of a
community or assemblage may correlate with increased ecosystem function,
via increases in trait diversity (Diane S Srivastava et al. 2012).
<!-- Cadotte et al. many, cavender-Bares ecol let review, Flynn et al., Mahereli and Klironomos, -->

Phylogenetic diversity measures have provided useful insights into
diverse plant communities (e.g. M. W. Cadotte et al. 2009; M. W.
Cadotte, Cardinale, and Oakley 2008; Godoy, Kraft, and Levine 2014), and
yet have rarely been applied to local assemblages of predators (L.
Bersier and Kehrli 2008; Naisbit et al. 2011). Many studies of
phylogenetic signal in predator traits focus on whole clades, rather
than local assemblages (e.g. *Anolis* lizards (Knouft et al. 2006),
warbler (Böhning-Gaese, Schuda, and Helbig 2003), treeboas (Henderson,
Pauers, and Colston 2013) and wasps (Budriene and Budrys 2004)) making
it difficult to connect these results to top-down effects at the scale
of a local community. While these clade-specific studies often find weak
evidence for phylogenetic signal in ecologically-relevant traits,
studies at the level of the whole biosphere (Gómez, Verdú, and Perfectti
2010; L. Bersier and Kehrli 2008) demonstrate that related organisms
often have similar interspecific interactions -- i.e. related predators
often consume similar prey. At the level of a community, the effect of a
predator assemblage will depend on both the spatial distribution and
trophic interactions of all predator species. Phylogeny may constrain
species distributions when distant relatives have distinct fundamental
niches, while close relatives are too similar to co-exist (Webb et al.
2002; Emerson and Gillespie 2008) -- for predators, this means that
similar taxa may not be able to occupy the same compartment of a food
web (Rezende et al. 2009). By correlating with these properties,
measures of phylogenetic diversity may allow us to predict the effect of
a predator assemblage on ecosystem functioning.

Within predator assemblages, there may be considerable variation in the
abundance and distribution of related taxa. If habitat patches are
variable, and if the fundamental niche of different organisms varies
with relatedness, then we would expect a phylogenetic signal to patterns
of occurrence. Specifically, if related predators share habitat
requirements, then we expect to find them in the same patches -- unless
their similarity precludes their co-occurrence. This relationship
between relatedness and distribution is he most common way in which
phylogenetic information is used in community ecology (Cavender-Bares et
al. 2009). In metacommunities -- i.e. when patches are connected by
dispersal -- variation in the composition of a local assemblage is also
determined by species dispersal into, and selection among, habitat
patches (M. A. Leibold et al. 2004; Howeth and Leibold 2010; Calcagno et
al. 2011). The effects of other predators can be important too: during
dispersal, assembly can be nonrandom if predators use cues of the
presence of prey, competitors, or intra-guild predators when colonizing
(Chase and Shulman 2009; McCauley et al. 2008; Diane S. Srivastava et
al. 2008). After colonization, patterns of co-occurrence may be further
modified by intraguild predation (L. P. Lounibos et al. 2008; Juliano
2009) or competition (Fincke, Yanoviak, and Hanschu 1997).

When predators co-occur, their feeding mode, diet breadth and
nutritional requirements will determine which prey they consume and the
potential total amount of predation. Because such morphological traits
may well be conserved over evolutionary time, the breadth of
differentiation (and hence resource use) may also show a phylogenetic
signal; this differentiation in resource use leads to complementarity
and greater overall ecosystem function (Finke and Snyder 2008). Among
predators, feeding mode is an important trait that is conserved within
lineages and determines a large part of diet breadth. For example, some
predators are gape-limited, and can swallow any prey smaller than their
mouth; these predators typically broaden their diet as they grow (e.g.,
larvae of Odonata) (E. Werner and Gilliam 1984). Such ontogenetic
changes in diet breadth can radically influence food web structure
(Moya-Laraño 2011). Other predators (e.g. Leeches, or Diptera:Tabanidae)
are "piercing-sucking" predators (Bay 1974) and may be more general
consumers at all life stages, since they can feed on larger prey
individuals even at very early predator life stages. In some cases,
predator diets may extend to include other predators leading to direct
negative interactions such as intraguild predation, which may also have
a phylogenetic signal (Pfennig 2000).

Predator interactions are common in many communities, and can lead to
nonlinearities which prevent the inference of predator effect from the
additive combination of individual predators (Sih, Englund, and Wooster
1998). For example, decreasing predator richness has been shown to
increase herbivory in a three-level kelp food web (Byrnes et al. 2006).
The complex traits that underlie such species interactions (particularly
"higher-order" interactions *sensu* Sih *et al.* (1998)) may also
correlate with phylogeny, leading to a higher correlation with community
processes than richness alone (M. W. Cadotte et al. 2009). For example,
phylogenetically diverse assemblages of mycorrhizae increased primary
productivity more than equally speciose treatments from the same lineage
(Maherali and Klironomos 2007). The effect of predators on prey may also
be non-additive, the result of indirect interactions either with prey
species or other predators: prey species may respond to the presence of
one predator by a behavioural shift that increases the predation by
another predator population (Carey and Wahl 2010). These trait-mediated
indirect effects (TMII, (E. Werner and Peacor 2003)) can be difficult to
predict; however phylogenetic relatedness has been suggested as a
possible means of predicting the combined effect of predators (Naisbit
et al. 2011)

We used a series of observations, lab feeding trials, and manipulative
field experiments to measure how the phylogenetic diversity of the
predator assemblage predicts community composition and ecosystem
function. We test three related hypotheses concerning co-occurance
patterns, diet similarity and top-down ecosystem effects of diverse
predators, using a natural mesocosm: the community of invertebrates
living within bromeliads. Bromeliads (Bromeliaceae) are epiphytic plants
native to the Neotropics. Many bromeliad species contain water, detritus
and a complex insect food web within their leaves; the decomposition of
this detritus supplies nutrients for the bromeliad (Benzing 2000). The
small size of these habitats permits direct manipulations of entire food
webs, manipulations which would be difficult in most natural systems.
Within this aquatic food web, damselfly larvae (e.g. *Leptagrion* spp.,
Odonata:Coenagrionidae) are important predators; their presence
dramatically alters community dynamics (e.g. decreases insect emergence
(Starzomski, Suen, and Srivastava 2010) and increases nutrient cycling
(Ngai and Srivastava 2006)).

1.  *species co-occurrence*: closely-related predators may occur
    together more frequently than less-related predators if there is a
    strong phylogenetic signal to habitat requirements. Alternatively,
    very closely related species may never co-occur because high overlap
    in ecological niches results in competitive exclusion.

2.  *diet similarity*: similarity in diet (as measured by feeding
    trials) decreases with phylogenetic distance if diet is
    phylogenetically conserved. Alternatively, closely related species
    may have evolved different diets to allow coexistence.

3.  *ecosystem-level effects*: Our experiments at the level of the whole
    habitat patch (i.e. a single bromeliad) allows us to examine direct
    and indirect effects of predator combinations.
    1.  monoculture treatments allow us to assess the effect of each
        predator on a complete prey community and ecosystem function.
        Phylogenetic similarity among predators is predicted to
        correlate with similarity in single-species effects if related
        predators share similar trophic interactions (e.g. predation
        rate, diet breadth)
    2.  By comparing treatments with pairs of predators to treatments
        which received a monoculture of each predator, we are able to
        estimate additive and nonadditive effects. If distance between
        species correlates with increasing difference, we will see an
        increase in these non-additive effects with phylogenetic
        distance. At the extreme, differences between predators may lead
        to IGP among predators.

Methods
-------

Site and Species
----------------

We conducted all observations and experiments in Parque Estadual da Ilha
do Cardoso (25<sup> ∘ </sup>03<sup>′</sup> S,
47<sup> ∘ </sup>53<sup>′</sup> W), a 22.5 ha island off the south coast
of São Paulo state, Brazil. We worked in a closed coastal forest
(restinga) the understory of which is mostly covered by **Quesnelia
arvensis** Mez. (Bromeliaceae), a large terrestrial bromeliad that
accumulates up to 2.8 L of rainwater in tanks formed by individual
leaves. Previous observational surveys of this work found more than 47
species of macroinvertebrates in these aquatic communities (Romero and
Srivastava 2010). This diversity encompasses multiple trophic groups.
Detritivores include various functional groups: shredders
(Diptera:Tipulidae, Trichoptera), scrapers (Coleoptera:Scirtidae), and
collectors (All Diptera:Chironomidae, Syrphidae, Psychodidae). Filter
feeders (Diptera:Culicidae) are supported by the microbial components of
the food web. A diverse predator assemblage consists of at least 3
species of damselfly larvae (*Leptagrion* spp.,
Zygoptera:Coenagrionidae), 2 species of predatory Horse Fly larvae
(Diptera:Tabanidae), and 2 species of leech (Hirudinidae). Other,
smaller-bodied predators present include predatory midge larvae
(Diptera:Chironomidae:Tanypodinae) and less abundant species such as
Dytiscid beetles.

### Phylogenetic distance, metabolic capacity and niche overlap

In 2008, we counted and measured macroinvertebrates in an observational
study of 25 bromeliads. Within this observational dataset, we identified
14 species as predators. These predators include species related at the
genus level -- e.g. *Bezzia* sp. (Diptera:Ceratopogonidae) with two
species and *Leptagrion* sp. (Odonata:Coenagrionidae) with three -- and
family level (three species of Tabanidae and two of Empididae, all
Diptera). Deeper divisions are also present: three families of Diptera
are represented by a single predator species each (Dolichopodidae,
Corethrellidae and Chironomidae) and the deepest taxonomic divide is
between all insects present and a species of predatory leech
(Annelida:Hirudinidae).

We obtained node age estimates for all seven internal nodes of the tree,
using `www.timetree.org`, an online database of published molecular time
estimates from the literature (Hedges, Dudley, and Kumar 2006). Most
dates came from only a single study; where multiple dates were found we
used the median estimate: Insecta--Hirudina (543 to 700 Mya, n=5
studies), Odonata--Tabanidae (151 to 543 Mya, n= studies) and
Tabanidae--Diptera (106.2 to 474 Mya, n=7 studies). Node age data was
available for all but the shallowest nodes of the tree, where either a
lack of taxonomic information (e.g. Tabanidae) or a lack of phylogenetic
study (e.g. *Leptagrion*) prevented more information from being
included. These branches were left as polytomies, and were all assigned
identical, arbitrary and short branch lengths (15 Mya).

Species co-occurrence is often measured in terms of non-random patterns
of species occurrence or abundance, but such measures will only be
poorly related to the functional effects of species when species differ
substantially in body size. Integrating the allometric relationship
between body size and feeding rate (Brown et al. 2004; Wilby, Villareal,
and Lan 2005) over all individuals of a species allows estimates of
"metabolic capacity", or the potential energy requirements of a species
(Diane S. Srivastava and Bell 2009). Metabolic capacity is equal to
individual body mass raised to the power of 0.69 (an insect-specific
exponent determined by Peters 19xx, (Chown et al. 2007)); this reflects
the nonlinearity of feeding rate on body size across many invertebrate
taxa. We used metabolic capacity to inform both our observational
results and our experimental design (details below), with the exception
of our feeding trial data. This is because the feeding trials were
intended to measure which prey our predators ate, rather than their
feeding rate (only the latter should scale with metabolic capacity).

Questions 1 and 2 above focus on niche overlap between predator species.
Question 1 hypothesizes that if diet is conserved, then diet similarity
declines with increased phylogenetic distance between a pair of
predators. Question 2 hypothesizes that if habitat preference is
phylogenetically conserved and competition between predators
unimportant, then co-occurence declines with increased phylogenetic
distance between a pair of predators. We evaluated both diet similarity
and co-occurence between predators using Pianka's index of niche overlap
(Pianka 1974):

\$O\_{kl}=\\dfrac{\\sum\_i\^n{p\_{il} p\_{ik}}}{\\sqrt{\\sum\_i\^n{p\_{il}\^2} \\sum\_i\^n{p\_{ik}\^2}}}\$

For each pair of predators, *p*<sub>*i**k*</sub> and
*p*<sub>*i**l*</sub> represent the preference of predator *k* (or *l*)
for resource or habitat *i*. In terms of predator co-occurrence, *n*=25
bromeliads surveyed in the observational dataset described below. In
terms of diet similarity, the number of resources (*n*) is defined as
the total number of prey species assayed with both predator taxa, and
preference is defined as the proportion of diet trials (see below) for
each predator-prey assay that resulted in prey mortality.

We quantified the effect of phylogenetic distance on each of
distributional and diet similarity. First, we calculated phylogenetic
distance between each pair of species, then fit several functions to
this relationship (linear, constant, and several appropriate nonlinear
functions). We compared these models using AIC, selected the best model,
and generated confidence intervals as appropriate (parametric or
bootstrap for linear and nonlinear, respectively).

### Distributional Similarity

We used the 2008 survey to examine patterns of co-occurrence among
predator taxa. Each bromeliad was dissected and washed to remove
invertebrates. The resulting water was filtered through two sieves (250
and 850 µm), which removed particulate organic matter without losing any
invertebrates. All invertebrates were counted and identified to the
lowest taxonomic level possible. The body length of all individuals was
measured, when possible for small and medium-sized taxa (\< 1cm final
instar) and always for large-bodied taxa (\> 1 cm final instar).

We calculated total metabolic capacity of each predator species in each
bromeliad by summing estimates of biomass for all individuals of the
same species within a plant. We used these values to calculate niche
overlap as measured by habitat occupancy. Overlap was calculated for
each pair of predators across all bromeliads, even if neither species
occurred there.

### Diet Similarity

We conducted 314 feeding trials of 10 predator taxa fed 14 prey taxa
between March and April 2011. We covered all potential predator-prey
pairs present in the experiment (described below), and attempted to
perform all other trials whenever possible. However, due to the rarity
of some taxa many predator-prey pairs were not possible; we tested 56
pairwise combinations. Most trials were replicated at least five times,
but the number of replicates for various combinations ranged from 1 to
11. We placed predators together with prey in a 50ml vial, with a leaf
or stick for substrate. The only exception was the tabanid larvae, which
we placed between two vertical surfaces to imitate the narrow space
found in bromeliad leaf axils, their preferred microhabitat. Generally
our trials contained a single predator and a single prey individual,
except in the case of very small prey (*Elpidium* sp.) or predators
(*Monopelopia* sp.) in which case we increased the density. We
replicated each combination up to five times where possible, and allowed
one day for predation to occur.

Again, we estimated overlap between different predators using Pianka's
index, using the proportion of trials which resulted in predation as our
measure of predator preference. Because not every predator/prey pair was
equally replicated, we weighted our analysis by the number of trials
conducted.

### Community effect experiment

Our experiment was created with two goals in mind: first, to measure the
major effects of each of these predators on their prey and ecosystem
function and second, to estimate the non- additive effects of pairwise
predator combinations. The strength of these non-additive effects could
then be related to the phylogenetic distance between a pair of predator
species.

We selected the four most common predators in this system, which also
created a range of relatedness: two congeneric damselflies (*Leptagrion
andromache* and *Leptagrion elongatum*), two insects (*L. elongatum* and
a Tabanid predatory fly), and two invertebrates (*L. elongatum* and
leeches). We used each of the pairs of predators just described, as well
as each species in monoculture, and a predator-free control (8
treatments, n=5). Combinations were substitutive, maintaining the same
amount of total predator metabolic capacity to isolate the effects of
predator behaviour and traits. Substitutive experiments often hold total
abundance constant, but when species differ substantially in body size -
as in this experiment - allometric effects of body size on feeding rate
can confound detection of effects based on trophic traits or species
interactions, and standardizing to community metabolic capacity is
preferred (Diane S. Srivastava and Bell 2009). This experiment allows
the estimation of the effect of each predator species (monoculture
treatments), as well as the detection of non- additive effects in
predator combinations.

In Feburary 2011, bromeliads between 90 and 200ml in capacity were
collected, thoroughly washed to remove organisms and detritus and soaked
for 12 hours in a tub of water. They were then hung for 48 hours to dry.
One bromeliad dissected after this procedure contained no insects.

Each bromeliad was supplied with dried leaves, simulating natural
detritus inputs from the canopy. We enriched these leaves with
<sup>15</sup>N by fertilizing five (Jabuticaba, *Plinia cauliflora*)
plants with 40ml pot<sup>-1</sup> day<sup>-1</sup> of 5g L<sup>-1</sup>
ammonium sulphate containing 10% atom excess of <sup>15</sup>N over 21
days. Whole leaves were then picked from plants and air-dried until
constant weight, and then soaked for three days and the water discarded.
About 1.5 g of leaves were placed in each bromeliad (1.5006g ± 0.0248).

Each bromeliad was stocked with a representative insect community. The
densities of each prey taxon were calculated from the 2008 observational
dataset, using data from bromeliads of similar size to those in our
experiment. All densities used were within the range of these calculated
abundances, and all experimental bromeliads received the same insect
community. Halfway through the experiment, insects were added to
bromeliads a second time to simulate the continuous oviposition that
characterizes the system. Throughout the experiment, all bromeliads were
enclosed with a mesh cage topped with a malaise trap and checked daily
for emergence of adults.

In this experiment we measured five response variables: production of
fine particulate organic matter (FPOM), decomposition of coarse
detritus, bromeliad growth, uptakle of detrital nitrogen into bromeliad
tissue, and survival of invertebrate prey (emerged adults + surviving
larvae). We analyzed each of these responses with ANOVA. For each pair
of predator species and each response type, we calculated the
non-additive effect as the difference between the response in bromeliads
with both predator species (n=5) and the mean response in bromeliads
with either one of these two predator species (n=5 for each predator
species). We generated bootstrap confidence intervals for these
nonadditive effects; confidence intervals which do not overlap zero
indicate a significant non-additive effect of a predator combination.

Results
-------

### Patterns of occurrence

Across all bromeliads, predator species differed widely in metabolic
capacity, from 4.5 × 10<sup>-4</sup> for a species of *Monopelopia* to
0.15 for large predatory flies in the family Tabanidae. Predators often
co-occurred in bromeliads (4.4 ± NA predator species per plant).
However, the niche overlap between the total metabolic capacity of pairs
of predators did not show any relationship with phylogenetic distance
between them (F<sub>1,89</sub>=2.3918, p=0.13).

### diet similarity

Among the most common predator taxa (i.e. those used in our experiment,
described below) the damselflies (*Leptagrion andromache* and
*Leptagrion elongatum*) showed the highest rates of prey consumption
(prey consumed in 94.3% and 67.7% of trials, respectively). All
predators showed a very generalist diet breadth, consuming nearly all
species offered to them. However, more phylogenetically distant
predators differed in their preference of prey species, as measured by
the niche overlap index (F<sub>1,\\ 26</sub>=5.98, p=0.022, regression
weighted by the number of prey species assayed.)

### Ecosystem-level effects and phylogenetic distance

In our manipulative experiment, we placed a standardized prey community
into bromeliads and measured five response variables. Predators had a
large effect on prey survivorship: on average all predator treatments
showed 51% lower prey emerging or surviving as larvae relative to the
predator-free control. Nitrogen transport to bromeliad leaves was
slightly decreased in bromeliads with predators relative to
predator-free controls (-11%), and was only higher than the control in
treatments including Tabanid predators. We found a similar pattern for
plant growth: on average, predators had a -18% effect on growth of
bromeliad leaves (leaf elongation in mm), though Tabanids seemed to
create a slight increase. The decomposition of coarse detritus and
production of fine organic matter showed no obvious pattern related to
the presence of predators.

Predator combinations tended to have a non-additive effect on our
response variables, even though we held total metabolic capacity
constant amongst all treatments. Approximately 14% more prey survived in
polyculture, on average, compared to all monocultures. Nitrogen uptake
increased by (8%) and bromeliad growth by (11%). Production of fine
particulate organic matter increased by 29% more when predators were
present in combination.

Our experimental design allows us to estimate the non-additive effect of
predator species pairs on whole communities of prey, and the functioning
of the bromeliad ecosystem. We used randomization tests to test the
hypothesis that increased phylogenetic distance between members of a
predator pair results in a greater magnitude of nonadditive effect. We
contrasted the differences of the mean individual predator treatments
from the control with the mean difference of their pairwise combination
from the control.

We found the greatest effect for prey survival: while effects of *L.
andromache* and *L. elongatum* in combination were quite similar to the
effect of either alone, when *L. elongatum* was placed in the same plant
as either a Tabanid larva or leeches, on average five more prey
individuals (18% of total prey community) survived till the end of the
experiment (Fig 3). This effect was smaller among the other variables,
most of which showed confidence intervals from the randomization test
which overlap zero.

### Figures

![Figure 1: Phylogenetic distance and predator co-occurrence. Each point
represents a pair of predator species. We calculated total metabolic
capacity for each predator species in each bromeliad, and then
calculated niche overlap between two predators using Pianka's index.
Solid line shows the best fit non-linear model (see Appendix) and dashed
lines show bootstrap 95%
quantiles.](./predatordiversity_files/figure-markdown_strict/FIG_metabolic_occurance_as_phylo.png)

![Figure 2: Diet overlap decreases with phylogenetic distance, based on
237 feeding trials with the 8 major predator taxa found in this system.
Diet similarity is calculated as Pianka's index of niche overlap. Solid
line shows the best fit non-linear model (see Appendix) and dashed lines
show bootstrap 95%
quantiles.](./predatordiversity_files/figure-markdown_strict/FIG_feeding_trial_as_phylo.png)

<table style="text-align:center"><tr><td colspan="6" style="border-bottom: 1px solid black"></td></tr><tr><td style="text-align:left"></td><td colspan="5">
<em>Dependent variable:</em>
</td></tr>
<tr><td></td><td colspan="5" style="border-bottom: 1px solid black"></td></tr>
<tr><td style="text-align:left"></td><td colspan="5">
val
</td></tr>
<tr><td style="text-align:left"></td><td>
total.surv
</td><td>
fine
</td><td>
decomp
</td><td>
growth
</td><td>
N
</td></tr>
<tr><td style="text-align:left"></td><td>
(1)
</td><td>
(2)
</td><td>
(3)
</td><td>
(4)
</td><td>
(5)
</td></tr>
<tr><td colspan="6" style="border-bottom: 1px solid black"></td></tr><tr><td style="text-align:left">
trt\_type.L
</td><td>
-4.384<sup>\*</sup>
</td><td>
0.015
</td><td>
0.007
</td><td>
-0.317
</td><td>
-2.418
</td></tr>
<tr><td style="text-align:left"></td><td>
(2.228)
</td><td>
(0.068)
</td><td>
(0.020)
</td><td>
(1.015)
</td><td>
(3.634)
</td></tr>
<tr><td style="text-align:left"></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td style="text-align:left">
trt\_type.Q
</td><td>
4.205<sup>\*</sup>
</td><td>
0.138<sup>\*</sup>
</td><td>
-0.011
</td><td>
0.664
</td><td>
4.638
</td></tr>
<tr><td style="text-align:left"></td><td>
(2.181)
</td><td>
(0.067)
</td><td>
(0.020)
</td><td>
(0.993)
</td><td>
(3.557)
</td></tr>
<tr><td style="text-align:left"></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td style="text-align:left">
Constant
</td><td>
9.583<sup>\*\*\*</sup>
</td><td>
0.577<sup>\*\*\*</sup>
</td><td>
0.339<sup>\*\*\*</sup>
</td><td>
4.175<sup>\*\*\*</sup>
</td><td>
49.280<sup>\*\*\*</sup>
</td></tr>
<tr><td style="text-align:left"></td><td>
(1.273)
</td><td>
(0.039)
</td><td>
(0.011)
</td><td>
(0.580)
</td><td>
(2.076)
</td></tr>
<tr><td style="text-align:left"></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td colspan="6" style="border-bottom: 1px solid black"></td></tr><tr><td style="text-align:left">
Observations
</td><td>
12
</td><td>
12
</td><td>
12
</td><td>
12
</td><td>
12
</td></tr>
<tr><td style="text-align:left">
R<sup>2</sup>
</td><td>
0.497
</td><td>
0.323
</td><td>
0.055
</td><td>
0.064
</td><td>
0.214
</td></tr>
<tr><td style="text-align:left">
Adjusted R<sup>2</sup>
</td><td>
0.385
</td><td>
0.172
</td><td>
-0.156
</td><td>
-0.143
</td><td>
0.039
</td></tr>
<tr><td style="text-align:left">
Residual Std. Error (df = 9)
</td><td>
4.315
</td><td>
0.132
</td><td>
0.039
</td><td>
1.965
</td><td>
7.037
</td></tr>
<tr><td style="text-align:left">
F Statistic (df = 2; 9)
</td><td>
4.450<sup>\*\*</sup>
</td><td>
2.143
</td><td>
0.260
</td><td>
0.310
</td><td>
1.226
</td></tr>
<tr><td colspan="6" style="border-bottom: 1px solid black"></td></tr><tr><td style="text-align:left">
<em>Note:</em>
</td><td colspan="5" style="text-align:right">
<sup>*</sup>p\<0.1; <sup>**</sup>p\<0.05; <sup>***</sup>p\<0.01
</td></tr>
</table>


![Figure 3: Combinations of predators beyond congenerics show a negative
non-additive effect on predation rate. Points represent the mean
difference between the means of two monocultures compared to the mean of
a polycultures. These values are presented as differences from (predator
free) controls; therefore zero indicates no significant non-additive
effect. Relative to control (no predator) plants, bromeliads containing
two predators which were not congeneric showed less
predation](./predatordiversity_files/figure-markdown_strict/FIG_PD_experiment_nonadditive.png)

Discussion
----------

References
----------

Bay, EC. 1974. “Predator-Prey Relationships Among Aquatic Insects.”
*Annual Review of Entomology* (19): 441–453.
<http://www.annualreviews.org/doi/pdf/10.1146/annurev.en.19.010174.002301>.

Benzing, David H. 2000. *Bromeliaceae: Profile of an Adaptive
Radiation*. Cambridge University Press.
<http://www.amazon.com/Bromeliaceae-Adaptive-Radiation-David-Benzing/dp/0521430313>.

Bersier, L, and P Kehrli. 2008. “The Signature of Phylogenetic
Constraints on Food-Web Structure.” *Ecological Complexity* 5 (2)
(June): 132–139.
doi:[10.1016/j.ecocom.2007.06.013](http://dx.doi.org/10.1016/j.ecocom.2007.06.013).
<http://linkinghub.elsevier.com/retrieve/pii/S1476945X08000093>.

Böhning-Gaese, K, M D Schuda, and a J Helbig. 2003. “Weak Phylogenetic
Effects on Ecological Niches of Sylvia Warblers.” *Journal of
Evolutionary Biology* 16 (5) (September): 956–65.
<http://www.ncbi.nlm.nih.gov/pubmed/14635911>.

Brown, James H., James F. Gillooly, Andrew P. Allen, Van M. Savage, and
Geoffrey B. West. 2004. “Toward a Metabolic Theory of Ecology.”
*Ecology* 85 (7) (July): 1771–1789.
doi:[10.1890/03-9000](http://dx.doi.org/10.1890/03-9000).
<http://www.esajournals.org/doi/abs/10.1890/03-9000>.

Budriene, A, and E Budrys. 2004. “Hunting Behaviour of Predatory Wasps
(Hymenoptera: Vespidae: Eumeninae): Is the Distribution of Stinging
Effort Phylogenetically Inherited or Dependent on the Prey.” *Annales de
La Société Entomologique de …* 40 (4): 259–268.
<http://www.tandfonline.com/doi/abs/10.1080/00379271.2004.10697424>.

Byrnes, Jarrett, John J Stachowicz, Kristin M Hultgren, a <span>Randall
Hughes</span>, Suzanne V Olyarnik, and Carol S Thornber. 2006. “Predator
Diversity Strengthens Trophic Cascades in Kelp Forests by Modifying
Herbivore Behaviour.” *Ecology Letters* 9 (1) (January): 61–71.
doi:[10.1111/j.1461-0248.2005.00842.x](http://dx.doi.org/10.1111/j.1461-0248.2005.00842.x).
<http://www.ncbi.nlm.nih.gov/pubmed/16958869>.

Cadotte, Marc W., Bradley J Cardinale, and Todd H Oakley. 2008.
“Evolutionary History and the Effect of Biodiversity on Plant
Productivity.” *Proceedings of the National Academy of Sciences of the
United States of America* 105 (44) (November): 17012–7.
doi:[10.1073/pnas.0805962105](http://dx.doi.org/10.1073/pnas.0805962105).
[http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2579369\\&tool=pmcentrez\\&rendertype=abstract](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2579369\&tool=pmcentrez\&rendertype=abstract).

Cadotte, Marc W., Jeannine Cavender-Bares, David Tilman, and Todd H
Oakley. 2009. “Using Phylogenetic, Functional and Trait Diversity to
Understand Patterns of Plant Community Productivity.” *PloS One* 4 (5)
(January): e5695.
doi:[10.1371/journal.pone.0005695](http://dx.doi.org/10.1371/journal.pone.0005695).
<http://www.ncbi.nlm.nih.gov/pubmed/19479086>.

Calcagno, Vincent, François Massol, Nicolas Mouquet, Philippe Jarne, and
Patrice David. 2011. “Constraints on Food Chain Length Arising from
Regional Metacommunity Dynamics.” *Proceedings. Biological Sciences /
The Royal Society* (iv) (March).
doi:[10.1098/rspb.2011.0112](http://dx.doi.org/10.1098/rspb.2011.0112).
<http://www.ncbi.nlm.nih.gov/pubmed/21367786>.

Carey, Michael P, and David H Wahl. 2010. “Interactions of Multiple
Predators with Different Foraging Modes in an Aquatic Food Web.”
*Oecologia* 162 (2) (February): 443–52.
doi:[10.1007/s00442-009-1461-3](http://dx.doi.org/10.1007/s00442-009-1461-3).
<http://www.ncbi.nlm.nih.gov/pubmed/19777265>.

Cavender-Bares, Jeannine, Kenneth H Kozak, Paul V a Fine, and Steven W
Kembel. 2009. “The Merging of Community Ecology and Phylogenetic
Biology.” *Ecology Letters* 12 (7) (July): 693–715.
doi:[10.1111/j.1461-0248.2009.01314.x](http://dx.doi.org/10.1111/j.1461-0248.2009.01314.x).
<http://www.ncbi.nlm.nih.gov/pubmed/19473217>.

Chase, Jonathan M., and Rachel S. Shulman. 2009. “Wetland Isolation
Facilitates Larval Mosquito Density Through the Reduction of Predators.”
*Ecological Entomology* 34 (6) (December): 741–747.
doi:[10.1111/j.1365-2311.2009.01128.x](http://dx.doi.org/10.1111/j.1365-2311.2009.01128.x).
<http://doi.wiley.com/10.1111/j.1365-2311.2009.01128.x>.

Chown, S. L., E. Marais, J. S. Terblanche, C. J. Klok, J. R. B. Lighton,
and T. M. Blackburn. 2007. “Scaling of Insect Metabolic Rate Is
Inconsistent with the Nutrient Supply Network Model.” *Functional
Ecology* 21 (2) (April): 282–290.
doi:[10.1111/j.1365-2435.2007.01245.x](http://dx.doi.org/10.1111/j.1365-2435.2007.01245.x).
<http://doi.wiley.com/10.1111/j.1365-2435.2007.01245.x>.

Emerson, Brent C, and Rosemary G Gillespie. 2008. “Phylogenetic Analysis
of Community Assembly and Structure over Space and Time.” *Trends in
Ecology & Evolution (Personal Edition)* 23 (11) (November): 619–30.
doi:[10.1016/j.tree.2008.07.005](http://dx.doi.org/10.1016/j.tree.2008.07.005).
<http://www.ncbi.nlm.nih.gov/pubmed/18823678>.

Estes, James a, John Terborgh, Justin S Brashares, Mary E Power, Joel
Berger, William J Bond, Stephen R Carpenter, et al. 2011. “Trophic
Downgrading of Planet Earth.” *Science (New York, N.Y.)* 333 (6040)
(July): 301–6.
doi:[10.1126/science.1205106](http://dx.doi.org/10.1126/science.1205106).
<http://www.ncbi.nlm.nih.gov/pubmed/21764740>.

Fincke, OM, SP Yanoviak, and RD Hanschu. 1997. “Predation by Odonates
Depresses Mosquito Abundance in Water-Filled Tree Holes in Panama.”
*Oecologia* 112: 244–253.
<http://link.springer.com/article/10.1007/s004420050307>.

Finke, Deborah L, and William E Snyder. 2008. “Niche Partitioning
Increases Resource Exploitation by Diverse Communities.” *Science (New
York, N.Y.)* 321 (5895) (September): 1488–90.
doi:[10.1126/science.1160854](http://dx.doi.org/10.1126/science.1160854).
<http://www.ncbi.nlm.nih.gov/pubmed/18787167>.

Godoy, Oscar, Nathan J. B. Kraft, and Jonathan M. Levine. 2014.
“Phylogenetic Relatedness and the Determinants of Competitive Outcomes.”
Edited by Jerome Chave. *Ecology Letters* (April): n/a–n/a.
doi:[10.1111/ele.12289](http://dx.doi.org/10.1111/ele.12289).
<http://doi.wiley.com/10.1111/ele.12289>.

Gómez, José M., Miguel Verdú, and Francisco Perfectti. 2010. “Ecological
Interactions Are Evolutionarily Conserved Across the Entire Tree of
Life.” *Nature* 465 (June) (June).
doi:[10.1038/nature09113](http://dx.doi.org/10.1038/nature09113).
<http://www.nature.com/doifinder/10.1038/nature09113>.

Griswold, Marcus W, and L Philip Lounibos. 2006. “Predator Identity and
Additive Effects in a Treehole Community.” *Ecology* 87 (4) (April):
987–95.
[http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1820834\\&tool=pmcentrez\\&rendertype=abstract](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1820834\&tool=pmcentrez\&rendertype=abstract).

Hedges, S Blair, Joel Dudley, and Sudhir Kumar. 2006. “TimeTree: a
Public Knowledge-Base of Divergence Times Among Organisms.”
*Bioinformatics (Oxford, England)* 22 (23) (December): 2971–2.
doi:[10.1093/bioinformatics/btl505](http://dx.doi.org/10.1093/bioinformatics/btl505).
<http://www.ncbi.nlm.nih.gov/pubmed/17021158>.

Henderson, Robert W., Michael J. Pauers, and Timothy J. Colston. 2013.
“On the Congruence of Morphology, Trophic Ecology, and Phylogeny in
Neotropical Treeboas (Squamata: Boidae: Corallus ).” *Biological Journal
of the Linnean Society* 109 (2) (June): 466–475.
doi:[10.1111/bij.12052](http://dx.doi.org/10.1111/bij.12052).
<http://doi.wiley.com/10.1111/bij.12052>.

Howeth, Jennifer G, and Mathew a Leibold. 2010. “Species Dispersal Rates
Alter Diversity and Ecosystem Stability in Pond Metacommunities.”
*Ecology* 91 (9) (September): 2727–41.
<http://www.ncbi.nlm.nih.gov/pubmed/20957966>.

Ives, Anthony R., Bradley J. Cardinale, and William E. Snyder. 2005. “A
Synthesis of Subdisciplines: predator-Prey Interactions, and
Biodiversity and Ecosystem Functioning.” *Ecology Letters* 8 (1)
(December): 102–116.
doi:[10.1111/j.1461-0248.2004.00698.x](http://dx.doi.org/10.1111/j.1461-0248.2004.00698.x).
<http://doi.wiley.com/10.1111/j.1461-0248.2004.00698.x>.

Juliano, SA. 2009. “Species Interactions Among Larval Mosquitoes:
context Dependence Across Habitat Gradients.” *Annual Review of
Entomology*: 37–56.
doi:[10.1146/annurev.ento.54.110807.090611.Species](http://dx.doi.org/10.1146/annurev.ento.54.110807.090611.Species).
<http://www.ncbi.nlm.nih.gov/pmc/articles/pmc2664081/>.

Knouft, JH, JB Losos, RE Glor, and JJ Kolbe. 2006. “Phylogenetic
Analysis of the Evolution of the Niche in Lizards of the Anolis Sagrei
Group.” *Ecology* 87 (7).
[http://www.esajournals.org/doi/abs/10.1890/0012-9658(2006)87[29:PAOTEO]2.0.CO;2 http://www.esajournals.org/doi/abs/10.1890/0012-9658(2006)87\\%5B29:PAOTEO\\%5D2.0.CO\\%3B2](http://www.esajournals.org/doi/abs/10.1890/0012-9658(2006)87[29:PAOTEO]2.0.CO;2 http://www.esajournals.org/doi/abs/10.1890/0012-9658(2006)87\%5B29:PAOTEO\%5D2.0.CO\%3B2).

Leibold, M A, M Holyoak, N Mouquet, P Amarasekare, Jonathan M. Chase, M
F Hoopes, R D Holt, et al. 2004. “The Metacommunity Concept: a Framework
for Multi-Scale Community Ecology.” *Ecology Letters* 7 (7): 601–613.
doi:[10.1111/j.1461-0248.2004.00608.x](http://dx.doi.org/10.1111/j.1461-0248.2004.00608.x).
<http://www.blackwell-synergy.com/links/doi/10.1111/j.1461-0248.2004.00608.x>.

Lounibos, L P, S Makhni, B W Alto, and B Kesavaraju. 2008. “Surplus
Killing by Predatory Larvae of Corethrella Appendiculata: Prepupal
Timing and Site-Specific Attack on Mosquito Prey.” *Journal of Insect
Behavior* 21 (2) (March): 47–54.
doi:[10.1007/s10905-007-9103-2](http://dx.doi.org/10.1007/s10905-007-9103-2).
[http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2600435\\&tool=pmcentrez\\&rendertype=abstract](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2600435\&tool=pmcentrez\&rendertype=abstract).

Maherali, Hafiz, and John N Klironomos. 2007. “Influence of Phylogeny on
Fungal Community Assembly and Ecosystem Functioning.” *Science (New
York, N.Y.)* 316 (5832) (June): 1746–8.
doi:[10.1126/science.1143082](http://dx.doi.org/10.1126/science.1143082).
<http://www.ncbi.nlm.nih.gov/pubmed/17588930>.

McCauley, Shannon J, Christopher J Davis, Rick a Relyea, Kerry L
Yurewicz, David K Skelly, and Earl E Werner. 2008. “Metacommunity
Patterns in Larval Odonates.” *Oecologia* 158 (2) (November): 329–42.
doi:[10.1007/s00442-008-1141-8](http://dx.doi.org/10.1007/s00442-008-1141-8).
<http://www.ncbi.nlm.nih.gov/pubmed/18781330>.

Moya-Laraño, Jordi. 2011. “Genetic Variation, Predator-Prey Interactions
and Food Web Structure.” *Philosophical Transactions of the Royal
Society of London. Series B, Biological Sciences* 366 (1569) (May):
1425–37.
doi:[10.1098/rstb.2010.0241](http://dx.doi.org/10.1098/rstb.2010.0241).
[http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3081571\\&tool=pmcentrez\\&rendertype=abstract](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3081571\&tool=pmcentrez\&rendertype=abstract).

Naisbit, Russell E, Patrik Kehrli, Rudolf P Rohr, and Louis-Félix
Bersier. 2011. “Phylogenetic Signal in Predator-Prey Body-Size
Relationships.” *Ecology* 92 (12) (December): 2183–9.
<http://www.ncbi.nlm.nih.gov/pubmed/22352156>.

Ngai, Jacqueline T., and Diane S. Srivastava. 2006. “Predators
Accelerate Nutrient Cycling in a Bromeliad Ecosystem.” *Science* 314
(5801): 963. <http://www.sciencemag.org/content/314/5801/963.short>.

Nyström, P, O Svensson, B Lardner, C Brönmark, and W Granéli. 2001. “The
Influence of Multiple Introduced Predators on a Littoral Pond
Community.” *Ecology* 82 (4): 1023–1039.
doi:[ http://dx.doi.org/10.1890/0012-9658(2001)082[1023:TIOMIP]2.0.CO;2](http://dx.doi.org/. http://dx.doi.org/10.1890/0012-9658(2001)082[1023:TIOMIP]2.0.CO;2).
[http://www.esajournals.org/doi/abs/10.1890/0012-9658(2001)082\\%5B1023:TIOMIP\\%5D2.0.CO\\%3B2](http://www.esajournals.org/doi/abs/10.1890/0012-9658(2001)082\%5B1023:TIOMIP\%5D2.0.CO\%3B2).

Pfennig, Dw. 2000. “Effect of Predator-Prey Phylogenetic Similarity on
the Fitness Consequences of Predation: A Trade-Off Between Nutrition and
Disease?” *The American Naturalist* 155 (3) (March): 335–345.
doi:[10.1086/303329](http://dx.doi.org/10.1086/303329).
<http://www.ncbi.nlm.nih.gov/pubmed/10718730>.

Pianka, ER. 1974. “Niche Overlap and Diffuse Competition.” *Proceedings
of the National Academy of …* 71 (5): 2141–2145.
<http://www.pnas.org/content/71/5/2141.short>.

Rezende, Enrico L, Eva M Albert, Miguel a Fortuna, and Jordi Bascompte.
2009. “Compartments in a Marine Food Web Associated with Phylogeny, Body
Mass, and Habitat Structure.” *Ecology Letters* 12 (8) (August): 779–88.
doi:[10.1111/j.1461-0248.2009.01327.x](http://dx.doi.org/10.1111/j.1461-0248.2009.01327.x).
<http://www.ncbi.nlm.nih.gov/pubmed/19490028>.

Romero, Gustavo Q, and Diane S Srivastava. 2010. “Food-Web Composition
Affects Cross-Ecosystem Interactions and Subsidies.” *The Journal of
Animal Ecology* 79 (5) (September): 1122–31.
doi:[10.1111/j.1365-2656.2010.01716.x](http://dx.doi.org/10.1111/j.1365-2656.2010.01716.x).
<http://www.ncbi.nlm.nih.gov/pubmed/20584097>.

Schmitz, Oswald J. 2007. “Predator Diversity and Trophic Interactions.”
*Ecology* 88 (10): 2415–2426.
<http://www.esajournals.org/doi/abs/10.1890/06-0937.1>.

Sih, Andrew, Goran Englund, and David Wooster. 1998. “Emergent Impacts
of Multiple Predators on Prey.” *Trends in Ecology & Evolution* 13 (9):
350–355.
<http://linkinghub.elsevier.com/retrieve/pii/S0169534798014372>.

Srivastava, Diane S, Marc W Cadotte, a Andrew M MacDonald, Robin G
Marushia, and Nicholas Mirotchnick. 2012. “Phylogenetic Diversity and
the Functioning of Ecosystems.” *Ecology Letters* 15 (7) (July): 637–48.
doi:[10.1111/j.1461-0248.2012.01795.x](http://dx.doi.org/10.1111/j.1461-0248.2012.01795.x).
<http://www.ncbi.nlm.nih.gov/pubmed/22583836>.

Srivastava, Diane S., and Thomas Bell. 2009. “Reducing Horizontal and
Vertical Diversity in a Foodweb Triggers Extinctions and Impacts
Functions.” *Ecology Letters* 12 (10) (October): 1016–28.
doi:[10.1111/j.1461-0248.2009.01357.x](http://dx.doi.org/10.1111/j.1461-0248.2009.01357.x).
<http://www.ncbi.nlm.nih.gov/pubmed/19702635>.

Srivastava, Diane S., M K Trzcinski, B a Richardson, and B Gilbert.
2008. “Why Are Predators More Sensitive to Habitat Size Than Their Prey?
Insights from Bromeliad Insect Food Webs.” *The American Naturalist* 172
(6) (December): 761–71.
doi:[10.1086/592868](http://dx.doi.org/10.1086/592868).
<http://www.ncbi.nlm.nih.gov/pubmed/18956977>.

Starzomski, Brian M., Daryl Suen, and Diane S. Srivastava. 2010.
“Predation and Facilitation Determine Chironomid Emergence in a
Bromeliad-Insect Food Web.” *Ecological Entomology* 35 (1) (February):
53–60.
doi:[10.1111/j.1365-2311.2009.01155.x](http://dx.doi.org/10.1111/j.1365-2311.2009.01155.x).
<http://doi.wiley.com/10.1111/j.1365-2311.2009.01155.x>.

Webb, Campbell O., David D. Ackerly, Mark a. McPeek, and Michael J.
Donoghue. 2002. “Phylogenies and Community Ecology.” *Annual Review of
Ecology and Systematics* 33 (1) (November): 475–505.
doi:[10.1146/annurev.ecolsys.33.010802.150448](http://dx.doi.org/10.1146/annurev.ecolsys.33.010802.150448).
<http://arjournals.annualreviews.org/doi/abs/10.1146/annurev.ecolsys.33.010802.150448>.

Werner, EE, and JF Gilliam. 1984. “The Ontogenetic Niche and Species
Interactions in Size-Structured Populations.” *Annual Review of Ecology
and Systematics* (15): 393–425. <http://www.jstor.org/stable/2096954>.

Werner, EE, and SD Peacor. 2003. “A Review of Trait-Mediated Indirect
Interactions in Ecological Communities.” *Ecology* 84 (5): 1083–1100.
[http://www.esajournals.org/doi/abs/10.1890/0012-9658(2003)084\\%5B1083:AROTII\\%5D2.0.CO\\%3B2](http://www.esajournals.org/doi/abs/10.1890/0012-9658(2003)084\%5B1083:AROTII\%5D2.0.CO\%3B2).

Wilby, A, SC Villareal, and LP Lan. 2005. “Functional Benefits of
Predator Species Diversity Depend on Prey Identity.” *Ecological …*:
497–501.
<http://onlinelibrary.wiley.com/doi/10.1111/j.0307-6946.2005.00717.x/full>.
